src/server/models/listItem/listItemEvent.ts

```diff
@@ -122,11 +122,11 @@ export const EVENTS = {
     },
   }),
 
-  [ListItemEvent.REMINDER]: (updatedJsonData = {}, reference?: string): EventCreate<"REMINDER"> => ({
+  [ListItemEvent.REMINDER]: (notes?: string[], reference?: string): EventCreate<"REMINDER"> => ({
```

Comment:
A reminder would not have updatedJsonData in it. 

I've added `notes`, incase implementation changes there is at least something descriptive to describe why this event was created. 

this will also included the response from govuk notify, which includes reference, ids etc.